### PR TITLE
Add rubygem-server_sent_events package

### PR DESCRIPTION
**Pull request by lzap**
The goal is to bring https://rubygems.org/gems/redfish_client in. This
is the dependency.

I used the script, no modifications.

* Nightly

### CODE REVIEW

**Comment by ekohl** [comment]
You can add the gem and all dependencies in a single PR, provided there are no build dependencies. This is the preferred approach since it means we can actually check repoclosure on all built RPMs together.

File: packages/foreman/rubygem-server_sent_events/rubygem-server_sent_events.spec [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.rubocop.yml
+%{gem_instdir}/.simplecov
```
Comment (ekohl) [inline packages/foreman/rubygem-server_sent_events/rubygem-server_sent_events.spec:77]:
```suggestion
%exclude %{gem_instdir}/.simplecov
```

**Comment by lzap** [comment]
Fixed.

**Comment by lzap** [comment]
I am not aware closing and deleting the branch myself. Reopening.

**Comment by lzap** [comment]
Rebased the conflict, can I get this one in?

**Comment by lzap** [comment]
There are no other dependencies actually, the next package will be https://rubygems.org/gems/redfish_client

Rebased. It's now in proxy EL7 and EL8.

**Review by evgeni — APPROVED** [review]

**Pull request merged**